app/translations/messages.en.yml

```
@@ -481,6 +481,7 @@ Entities:
       synchronizeGroup: "Also synchronize groups"
       ldapGroupNameField: "LDAP field for group name (ex : sn)"
       ldapEmailField: "LDAP field for email"
+      ldapAliasField: "LDAP field for alias (if different of email)"
```

I think it should be (I'm not entirely sure :))

```suggestion
      ldapAliasField: "LDAP field for alias (if different from email)"
```